src/pages/patientView/pathologyReport/PathologyReport.tsx

```diff
@@ -35,26 +36,25 @@ export default class PathologyReport extends React.Component<IPathologyReportPro
 
     }
 
-    shouldComponentUpdate(nextProps: IPathologyReportProps){
-        return nextProps === this.props;
-    }
+    // shouldComponentUpdate(nextProps: IPathologyReportProps){
+    //     return nextProps === this.props;
+    // }
 
     handleSelection(){
         this.setState({ pdfUrl:this.buildPDFUrl(this.pdfSelectList.options[this.pdfSelectList.selectedIndex].value) });
     }
 
     render(){
 
-        return (<div>
+        return (<div style={{position: "relative"}}>
```

Comment:
what's reason for position:relative?

Reply:
hmm, there is no special reason for that. I can clean the unneeded style.